rest/admin_api.go

```diff
@@ -32,7 +32,7 @@ const kDefaultDBOnlineDelay = 0
 // "Create" a database (actually just register an existing bucket)
 func (h *handler) handleCreateDB() error {
 	h.assertAdminOnly()
-	dbName := h.PathVar("newdb")
+	dbName := h.PathVar("MD_newdb")
```

Comment:
are there any alternatives to tacking on `MD_` to the string?  For example, would it be possible to do some sort of wrapper approach like:

```
dbName := h.PathVar(MD("newdb"))
```

or since `MD()` is already used, what about something like:

```
dbName := h.RedactPathVar("newdb", redact.MD)
```

Reply:
I like this approach. It's a lot easier to change in the future, even if the wrapper function is still just prepending a value to the name.